tests/fixtures/large_bitcoin_signature_request.json

```diff
@@ -1,334 +1,290 @@
 {
 
-    "redeem_script": "522102a567420d0ecb8ae1ac3794a6f901c9fdfa63a24476b8889d4c483dc7975f4ab321034a7496c358ee925043f5859bf7a191566fa070e3b75aac3f3bde6a670a8b6a8e2103666946ad4ff2b8c5c1ed6c8f5dd7f28769820cf35061ad5e02a45c5f05d54a1853ae",
-
-    "bip32_path": "m/45'/1'/120'/20/26",
-
     "inputs": [
+      [
```

Comment:
this is the most utxos that could fit in a request, might make sense to test with all different input groups to have that limit.